1.0/docs/devguide/local-dom.md

```diff
@@ -8,7 +8,7 @@ subtitle: Developer guide
 
 {% include toc.html %}
 
-We call the DOM that an element is in charge of creating and managing its _local
+We call the DOM an element that is in charge of creating and managing its _local
```

Comment:
+1 for Eric's suggestion of `The DOM that an element creates and/or manages is called its local DOM.`